languages/ruby/reference/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,61 @@
-# How to implement an Ruby concept exercise
+# How to implement a Ruby concept exercise
 
-TODO: describe how to implement a concept exercise for the Ruby track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes how to implement a concept exercise for the Ruby track. As this document is generic, `<NAME>` will be use as the placeholder for the name of the exercise in underscore-case (e.g. `string_interpolation`)
 
-[csharp-implementing]: ../../csharp/reference/implementing-a-concept-exercise.md
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue. Having done this, please read the [Ruby concept exercises introduction][concept-exercises].
+
+To implement a concept exercise, the following files must be created:
+
+<pre>
+languages
+└── ruby
+    └── exercises
+        └── concept
+            └── &lt;NAME&gt;
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
+                ├── .meta
+                |   |── config.json
+                |   |── design.md
+                |   └── example.rb
+                ├── &lt;NAME&gt;.rb
+                └── &lt;NAME&gt;_test.rb
+</pre>
+
+## Step 1: adding track-specific files
+
+These files are specific to the Ruby track:
+
+- `<NAME>.rb`. the stub implementation file, which is the starting point for students to work on the exercise.
```

Comment:
This uses the literal greater than, and less than symbols, while the lines above (14, 24 and 25) use the HTML entities, can we get this uniform (perhaps with the literal characters)?

Reply:
👍 I also prefer literal characters. I copied the tree structure form the C# track and now I'm wondering if there is any reason why HTML entities are used in there. For example, when the tree is meant to be displayed in a web component which needs HTML entities.

Reply:
I would guess that if there were literals and entities mixed in the original, and it hasn't caused programmatic problems and so it stayed.